src/me/vedang/clj_fdb/core.clj

```diff
@@ -1,11 +1,12 @@
 (ns me.vedang.clj-fdb.core
   (:refer-clojure :exclude [get set range])
   (:require [me.vedang.clj-fdb.impl :as fimpl]
-            [me.vedang.clj-fdb.subspace.subspace :as fsub]
+            [me.vedang.clj-fdb.key-selector :as sut]
```

Comment:
I'm not in favor of calling a namespace `sut` . Please use a different namespace shortname. I think `fks` is a better shortname

Reply:
I am mainly using `sut` because you used it in your test ns. https://github.com/vedang/clj_fdb/blob/be8530011fdb4552f5b313b21416bdbe989654a2/test/me/vedang/clj_fdb/key_selector_test.clj#L5

Reply:
Done.